### DISCUSSION TITLE
How to integrate with trpc-shield

### DISCUSSION THREAD

**Discussion by lewebsimple** [wobsoriano/trpc-nuxt#98] [Q&A] [answered]
Thanks for the great package, I'm wondering if example usage with trpc-shield is planned in the future ? The documentation provides some examples but it would be nice to also have an example using trpc-shield.

**Comment by RayGuo-ergou** [comment, marked as answer]
`trpc-shield` is just an extension for `trpc middleware`. I reckon that you can simply use `trpc-shield` example.
